proto/cmp/services/activity/v4/short_list.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package cmp.services.activity.v4;
+
+import "buf/validate/validate.proto";
+import "cmp/services/activity/v4/activity_types.proto";
+import "cmp/types/v4/common.proto";
+import "google/protobuf/timestamp.proto";
+
+// Request a list of modified activities.
+message ActivityProductShortListRequest {
+  // Common request header used in every request.
+  cmp.types.v4.RequestHeader header = 1 [(buf.validate.field).required = true];
+
+  // Only respond with the products that are modified after this timestamp
+  //
+  // The "modified_after" parameter is used to identify updated products from
+  // Suppliers.
+  google.protobuf.Timestamp modified_after = 2;
+}
+
+// Respond a list of modified activities.
+message ActivityProductShortListResponse {
+  // Common response header used in every response.
+  cmp.types.v4.ResponseHeader header = 1 [(buf.validate.field).required = true];
+
+  // Supplier codes and status for the products that are modified after the timestamp
+  // in the `ActivityProductShortListRequest`. If there are no changes, the list
+  // will be empty.
+  repeated cmp.services.activity.v4.ActivityShortListItem activity_short_list_items = 2;
+}
+
+// Activity product short list service definition. This is an alternative
+// to `ActivityProductListService`.
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/activity/v4/list.proto.dot.xs.svg)
+//
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/activity/v4/list.proto.dot.svg)
+//
+/// @custom:cmp-service type:product routing:p2p on-chain:false
```

Comment:
```suggestion
// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/activity/v4/short_list.proto.dot.xs.svg)
//
// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/activity/v4/short_list.proto.dot.svg)
//
/// @custom:cmp-service type:product routing:p2p on-chain:false
```
This should be fixed to include the correct name.

Also I see this is wrong in the accommodation short list too.